tests/unit/htdocs/Frameworks/art/FunctionsIniTest.php

```
@@ -281,6 +281,69 @@ public function frameworksRootPathIsDefined(): void
+        $this->assertTrue(
+            function_exists('mod_loadRenderer'),
+            'mod_loadRenderer() should be defined in functions.ini.php'
+        );
+    }
+
+    #[Test]
+    public function modLoadRendererReturnsInstanceFromStaticCall(): void
+    {
+        // Define a test renderer class with a static instance() method
+        if (!class_exists('TestmodWidgetRenderer', false)) {
+            eval('class TestmodWidgetRenderer { private static $inst; public static function instance() { if (!self::$inst) { self::$inst = new self(); } return self::$inst; } }');
```

PHPMD (2.15.0)

[error] The method modLoadRendererReturnsInstanceFromStaticCall() contains an eval expression. (undefined)

(EvalExpression)